test/test-execute/exec-standardoutput-append.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Test for StandardOutput=append:
+
+[Service]
+ExecStartPre=sh -c 'printf hello\\\\n > /tmp/test-exec-standardoutput-output'
```

Comment:
You can also let systemd expand the newlines on itself... If you use a bare \n here it will do that. As long as the string looks quoted for the shell itself, you'll be fine.

See also an example here: https://github.com/systemd/systemd/blob/v239/test/TEST-04-JOURNAL/test.sh#L38

This should work I think:

```
ExecStartPre=sh -c 'printf "hello\n" > /tmp/test-exec-standardoutput-output'
ExecStartPre=sh -c 'printf "hello\nhello\n" > /tmp/test-exec-standardoutput-expected'
```

Or just use `echo` to avoid one of the `\n`s:

```
ExecStartPre=sh -c 'echo "hello" > /tmp/test-exec-standardoutput-output'
ExecStartPre=sh -c 'echo "hello\nhello" > /tmp/test-exec-standardoutput-expected'
```

(Another option is to use command grouping on the second line, to run two separate "echo" commands and still keep a single redirection):

```
ExecStartPre=sh -c 'echo "hello" > /tmp/test-exec-standardoutput-output'
ExecStartPre=sh -c '{ echo "hello"; echo "hello"; } > /tmp/test-exec-standardoutput-expected'
```

I think I like the middle one most... It's a bit annoying that the `\n` is expanded by systemd while the rest is shell, but oh well that's systemd unit parsing for you :-)

Cheers,
Filipe